apps/web/src/css/global.css

```
@@ -1,6 +1,42 @@
 @import 'tailwindcss';
 @plugin '@tailwindcss/typography';
 
+@font-face {
+  font-family: 'AirbnbCerealLight';
```

Stylelint (17.6.0)

[error] Unexpected quotes around "AirbnbCerealLight" (font-family-name-quotes)

(font-family-name-quotes)